src/util/amount.rs

```diff
@@ -608,9 +608,10 @@ impl SignedAmount {
         if satoshi > i64::max_value() as u64 {
             return Err(ParseAmountError::TooBig);
         }
-        Ok(match negative {
-            true => SignedAmount(-1 * satoshi as i64),
-            false => SignedAmount(satoshi as i64),
+        Ok(if negative {
+            SignedAmount(-(satoshi as i64))
+        } else {
+            SignedAmount(satoshi as i64)
```

Comment:
I don't feel strongly about this, but why is 5 lines better than 4?

Reply:
supposedly matching over a bool is less readable. I think that sometimes match can be more readable, don't have strong feelings about this too https://rust-lang.github.io/rust-clippy/master/index.html#match_bool